### PR TITLE
[Feature] code base supports env variable

### PR DESCRIPTION
**Pull request by rueshyna**
code base supports env variable

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -30,15 +39,29 @@ jobs:
...
       - name: Build and push release version
-        uses: docker/build-push-action@v3
+        if: ${{ github.ref == 'refs/heads/release'}}
+        uses: docker/build-push-action@v4
+        with:
+          file: ./Dockerfile
+          push: true
+          tags: |
+            ghcr.io/marigold-dev/multisig:stable
+            ghcr.io/marigold-dev/multisig:${{ steps.tag_version.outputs.new_version}}-${{ steps.vars.outputs.sha_short }}-release
```
Comment (rueshyna) [inline .github/workflows/docker.yml:52]:
```suggestion
            ghcr.io/marigold-dev/multisig:${{ steps.tag_version.outputs.new_version}}-${{ steps.vars.outputs.sha_short }}-release-prod
```
Is it ok to add that?

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -30,15 +39,29 @@ jobs:
...
+      - name: Build and push staging version
+        if: ${{ github.ref != 'refs/heads/release'}}
+        uses: docker/build-push-action@v4
         with:
           file: ./Dockerfile
           push: true
           tags: |
             ghcr.io/marigold-dev/multisig:stable
-            ghcr.io/marigold-dev/multisig:${{ steps.vars.outputs.date }}-${{ steps.vars.outputs.sha_short }}-release
+            ghcr.io/marigold-dev/multisig:${{ steps.tag_version.outputs.new_version}}-${{ steps.vars.outputs.sha_short }}-release
```
Comment (rueshyna) [inline .github/workflows/docker.yml:64]:
```suggestion
            ghcr.io/marigold-dev/multisig:${{ steps.tag_version.outputs.new_version}}-${{ steps.vars.outputs.sha_short }}-release-staging
```

**Review by rueshyna — COMMENTED** [review]
others looks good to me

**Comment by rueshyna** [comment]
Don't forget the text `Ghostnet`on `components/navbar.tsx`

**Comment by JulesGuesnon** [comment]
So for the CI part, @Laucans told me he'll have look to make sure it works well with our infra

**Comment by rueshyna** [comment]
The change looks good to me. Let's wait for @Laucans's review

File: .github/workflows/docker.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (rueshyna) [reply .github/workflows/docker.yml]: Thanks @cyrilevos . I saw @Laucans PR, I removed workflows from this PR.

**Comment by rueshyna** [comment]
rebased this PR and ci work move to https://github.com/marigold-dev/Multisig-ui/pull/27

**Pull request merged**